syllabus/5-Substrate/1-Intro-to-Substrate_Slides.md

```diff
@@ -496,6 +472,37 @@ block `[0, .., N-1]`.
 
 ofc, changing the Wasm code cannot be done by anyone. That's up to governance.
 
+---v
+
+## The Runtime
+
+<div>
+
+- Runtime -> **Application Logic**.
+
+</div>
+<!-- .element: class="fragment" -->
+<div>
+
+- A _fancy_ term: Runtime -> **State Transition Function**.
```

Comment:
I've talked extensively about state machines in module 3. Might be nice to point out that the runtime is is a state machine. Or if you really prefer the term "state transition function" maybe at least an oral mention that "this is what Joshy called a state machine".

Reply:
Added to both slides and speaker notes.

Reply:
The correct statement seems to be:

A blockchain is a state machine. The runtime is the STF, which dictates how we move from one state to the other.

Reply:
I really wish you and shawn could see my module at least once. It feels impossible to tell a cohesive story when all we have is little github comments.